.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,21 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+## Don't delete anything without give instructions for you to delete it and don't change any thing without words those you have to change.
```

Comment:
This should have been,
Dont delete anything without explicit instruction s from a maintainer.